qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -141,6 +143,14 @@ def experiments(
             group: Filter by hub and group. `hub` must also be specified if `group` is.
             project: Filter by hub, group, and project. `hub` and `group` must also be
                 specified if `project` is.
+            exclude_public: If False (the default), experiments with a `public`
+                `share_level` may be returned. If True, experiments with a `public`
+                `share_level` will not be returned. Cannot be True if `public_only`
+                is True.
+            public_only: If False (the default), experiments with a `public`
+                `share_level` may be returned. If True, only experiments with a
+                `public` `share_level` will be returned. Cannot be True if
+                `exclude_public` is True.
```

Comment:
some nit pick on docstring: 
1. We use double backticks for code text as they're rendered as monospaced text. Single backtick is for parameters of this method (rendered as italic).
2. `True` and `False` also need to be quoted with double backticks, if they refer to boolean `True` and `False` (instead of the English words)
3. I'd do a short explanation on what "a public share_level" means, so something like
```
            exclude_public: If ``False`` (the default), experiments with ``share_level=public``
                (that is, experiments visible to all users)  may be returned. If ``True``, public experiments are excluded. 
                Cannot be ``True`` if `public_only` is ``True``.
```